components/AddMedication.jsx

```diff
@@ -22,6 +22,8 @@ export default function AddMedication() {
     if (Period === 'hour') increment = 3600000;
     if (Period === 'day') increment = 86400000;
     if (Period === 'week') increment = 604800000;
+    const firstDose =
+      Date.UTC(2022, Month, FormDate, parseInt(Time.substring(0, 2)), parseInt(Time.slice(-2)), 0) / 1000;
```

Comment:
it would be better to use `new Date().getFullYear()` to get the current year rather than hard coding it.